webui/react/src/hooks/useSettings.ts

```diff
@@ -31,7 +31,7 @@ interface UserSettingUpdate extends UpdateUserSettingParams {
   userId: number;
 }
 
-export type UpdateSettings = (updates: Settings, shouldPush?: boolean) => Promise<void>;
```

Comment:
i think we want `shouldPush` or something similar to filter out some query params. For example, if user select some rows in experiment table, `row` params are pushed. However, we want to avoid having `row` in params

Reply:
`shouldPush` just determined whether to use replaceState or pushState.

Reply:
https://github.com/determined-ai/determined/blob/master/webui/react/src/hooks/useSettings.ts#L308

Reply:
thanks. so the regression of `row` values in query params comes from other place. i think we talked about this before, and decide not to show `row` and some values in query params.

Reply:
That's part of the config.

Reply:
@gt2345 could you add `skipUrlEncoding: true,` for [row](https://github.com/determined-ai/determined/blob/master/webui/react/src/pages/ExperimentList.settings.ts#L135)

Reply:
What is the context behind changing the behavior here?

Reply:
<img width="1331" alt="image" src="https://user-images.githubusercontent.com/109113616/227061571-97bb03eb-c6a5-4db2-8ad0-ff1ea91969ed.png">

to avoid having this. we used to show it, but decided not to show it. but somehow its visible now

Reply:
Just to confirm, we want to hide row for all our tables or just the experiment list table?

Reply:
i think all of them, but let me clarify with Claire. i personally think we dont even need to use useSettings for row tho. 
@ClaireNeveu do we wanna hide all `row` param in URL?